src/Message.php

```diff
@@ -218,7 +225,11 @@ public function getBodyText()
         }
 
         // If message has no parts, return content of message itself.
-        return $this->getDecodedContent($this->keepUnseen);
+        if ($this->getSubtype() != 'HTML') {
```

Comment:
leading from the existing examples in the code

```        
        if ($this->getSubtype() == 'PLAIN') {
            return $this->getDecodedContent($this->keepUnseen);
        }
```

would surely be a more consistent way of expressing?